scripts/linters/test_files/invalid_merge_conflict.py

```diff
@@ -20,8 +20,4 @@
 
 from __future__ import annotations
 
-<<<<<<< HEAD
```

Comment:
Why are you deleting this?

Please proofread your changes before asking for reviews (i.e. do a self-review) to make sure you aren't including any changes that you aren't meaning to.

Reply:
Fixed